hvac/v1/__init__.py

```diff
@@ -265,6 +265,23 @@ def write(self, path, wrap_ttl=None, **kwargs):
         """
         return self._adapter.post(f"/v1/{path}", json=kwargs, wrap_ttl=wrap_ttl)
 
+    def write_fix_path(self, path, data={}, wrap_ttl=None):
+        """Same as write() method but fix issue #133 (you can pass `path` key in POST data)
```

Comment:
```suggestion
        """Write data to a path. Similar to write() without restrictions on data keys.
```

I also suggest that we add a description to `write` that suggests the use of `write_data`. I can't make this a real "suggestion" in GitHub because the line wasn't changed (line 255), but here's a draft:

> Write data to a path. Because this method uses kwargs for the data to write, "path" and "wrap_ttl" data keys cannot be used. If these names are needed, or if the key names are not known at design time, consider using the `write_data` method.

Reply:
Yes I agree with the idea :)